TUTORIAL.rst

```diff
@@ -0,0 +1,112 @@
+PDFmorph Tutorial
+#################
+
+Welcome! This will be a quick tutorial to accquaint users with PDFmorph
+and some of what it can do. 
+
+As we described in the README and installation instructions, please make
+sure that you are familiar with working with your command line terminal
+before using this application.
+
+Before you've started this tutorial, please ensure that you've installed
+all necessary software and dependencies.
+
+PDFmorph Example
+----------------
+
+	1. Open your Terminal or Command Prompt.
+
+	2. It it's not active already, activate your PDFmorph-equipped 
+	   conda environment by typing in ::
+	
+		source activate <pdfmorph_env>
+
+	   on Linux or ``activate <pdfmorph_env>`` on Windows.
+
+		* If you need to list your available conda environments,
+		  run the command ``conda info --envs`` or 
+		  ``conda env list``
+
+	3. Change directories with the ``cd`` command to a directory
+	   containing PDF files. For instance, I will use the file
+	   "ni_qmax25.cgr", found `here<https://github.com/diffpy/diffpy.pdfmorph/tree/master/tests/testdata>`_,
+	   so I will run ``cd ~/diffpy.pdfmorph/diffpy/pdfmorph/tests/testdata``
+
+	4. Using this file for an example (you can download it from the
+	   link above!), run the following command ::
+
+		pdfmorph ni_qmax25.cgr ni_qmax25.cgr
+
+	   This should produce an image like this ::
+
+		.. figure:: doc/images/morph_ex1.png
+			:align: center
+
+	   A few things to note:
+
+		* The green line (the difference curve) is flat because
+		  in this example, we used the same PDF twice (so, there
+		  is no difference)
+
+		* The red curve is the taget PDF, and the blue circles
```

Comment:
We usually use red line for model and blue dots for data (and green line offset for difference) in our normal PDF papers where we fit models to data.  In the PDFmorph context they are usually both data, so we should think about what makjes the most sense, but if we stick with red line and blue circles, the target should be the blue and the morph should be the red.

For the figure, ticks inside the box please.  Also, no legend, though reproducing the Rw in the panel I think is a would be a nice touch.

Reply:
I would start with a few words about what PDFmorph is doing (more or less cut and paste from the README), then type, "in general you run PDFmorph with the following command ``pdfmorph <target PDF> <morph PDF>`` "(check I have it the right way around)  " what happened here is we made the same PDF file to be target and morph ..."  tell the story.